### PR TITLE
Admin invoices

### PR DESCRIPTION
**Pull request by SMcPhee19**
Finishes User stories 32-36

I refactored User story 36 after talking to Chris. I way over coded the update method, so I have refactored that by getting rid of a few lines of code. I have also changed the way it is tested for better, more robust testing.

### CODE REVIEW

File: app/controllers/admin/invoices_controller.rb [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
   end
 
   def show
+    @invoice = Invoice.find(params[:id])
+  end
+
+  def update
+    invoice = Invoice.find(params[:id])
+    invoice.update(invoice_params)
+    redirect_to admin_invoice_path(invoice.id)
```
Comment (JCBeldo) [inline app/controllers/admin/invoices_controller.rb:13]: What happened to the conditional in this update or was that only for merchant/invoices?

Comment (reidsmiller) [reply app/controllers/admin/invoices_controller.rb:13]: It's just not in here. There's an update conditional for admin merchants. This is admin invoices, so there might be one in merchant invoices

Comment (SMcPhee19) [reply app/controllers/admin/invoices_controller.rb:13]: That conditional was doing something that was already being done on line 12. I had Chris take a look at it office hours and said that we over coded it.

Comment (JCBeldo) [reply app/controllers/admin/invoices_controller.rb:13]: Oh, awesome! Good to know.  I still need to do it though, I think, because i'm nested.

**Review by reidsmiller — APPROVED** [review]
With the fixed merge issues I think this should be good now. We'll need to check to see what happens when it is merged into main.

**Review by AlejandroLopez1992 — APPROVED** [review]

**Pull request merged**